packages/studio-base/package.json

```
@@ -190,7 +190,7 @@
     "ts-loader": "9.4.2",
     "tss-react": "4.7.7",
     "ttypescript": "1.5.15",
```

```suggestion
    "ttsc": "0.0.5",
```
I think changing this would work nicely.

---

These libraries support TS@5 and use the
and provides the same functionality as ttypescript.

---

This looks like a fork of ttypescript. Is there a PR for ttypescript that contributes these changes back so we can stay on ttypescript?

---

Even with the PR, cevek has been quiet.
That's why we posted the new library.

PS.
The recommended library is ts-patch, but it doesn't support TS5 yet.
It looks like it will be supported soon.
The usage is pretty much the same for ttypescript, ttsc, and ts-patch.